docker-app/qfieldcloud/core/models.py

```diff
@@ -1218,6 +1239,58 @@ class Meta:
         default=False,
     )
 
+    def get_localized_datasets_project(self) -> Optional[Project]:
```

Comment:
```suggestion
    def get_localized_datasets_project(self) -> Project | None:
```
Unless Python version restricts us, as in `(lib)qfieldsync`, we always prefer to use the more modern syntax for type definition.

Reply:
Updated!